src/js/index.js

```diff
@@ -104,3 +105,26 @@ window.addEventListener('load', () => {
 
 	cascade.call(window, true);
 });
+
+function setTheme(preference) {
```

Comment:
There is lot of repetitions in this function and the following one, maybe we can do something to get some more generic theme switching engine.

Reply:
Ok got it.

Reply:
I haven't been able to think of something that doesn't make it unecessarily complicated :/